src/Events/Integrations/Plugins/Elementor/Widgets/Traits/Event_Query.php

```diff
@@ -312,8 +313,9 @@ public function get_id_from_repository(): ?int {
 	 */
 	protected function get_event_id(): ?int {
 		$event_id = get_the_ID();
+		$settings = $this->get_settings();
 
-		if ( ! empty( $this->get_data( 'settings' ) ) ) {
+		if ( ! empty( $settings ) ) {
```

Comment:
```suggestion
		if ( empty( $settings ) ) {
```
No need to recreate it - the original check was to try and prevent some fatals when called early